js/form.js

```diff
@@ -1,7 +1,30 @@
 const contactForm = document.querySelector('.form-action');
 const error = document.querySelector('small');
+const formInput = document.querySelectorAll('.form-data');
 const email = document.querySelector('#email');
 
+let form = {
+  fullName: formInput[0].value,
+  email: formInput[1].value,
+  comment: formInput[2].value,
+};
+
+// add data base to object after reloading
+form = JSON.parse(localStorage.getItem('form'));
+// add object properties input fields after reload
+formInput[0].value = form.fullName;
+formInput[1].value = form.email;
+formInput[2].value = form.comment;
```

Comment:
- [ ] Hi @mohisa302 nice work, however currently no data is being preserved to the localstorage once the form is filled, Instead there is an error in the code, I  think it is because you are not checking if there is formdata in the localstorage initially, and instead directly parsing the localstorage data to your form variable see this error :point_down: 
![image](https://user-images.githubusercontent.com/97521557/209222401-d6005b20-baeb-43b8-a6b3-23072c29dbfe.png)

you also do not have a reload event like 'DOMContentLoaded" which will take the data from the localstorage if it is not empty, and will parse that data and fill the form with that data.
We can fix this by adding a check and a reload event like this below
```
const data = () => {
  formData = JSON.parse(localStorage.getItem('form'))
 if (formData) {
// add object properties input fields after reload
formInput[0].value = formData.fullName;
formInput[1].value = formData.email;
formInput[2].value = formData.comment;
}
}

window.addEventListener('DOMContentLoaded', data)
```